RadixWallet/Clients/OverlayWindowClient/OverlayWindowClient+Interface.swift

```diff
@@ -89,8 +89,15 @@ extension OverlayWindowClient {
 			// FIXME: Strings
 			public static let updated = Self(text: "Updated")
 
+			// FIXME: Strings
 			public static let seedPhraseImported = Self(text: "Seed Phrase Imported")
 
+			// FIXME: Strings
+			public static let accountHidden = Self(text: "Account hidden")
+
+			// FIXME: Strings
+			public static let personaHidden = Self(text: "Persona hidden")
+
```

Comment:
I moved `updated` to the place where it was used, it seems that all of these are single use things, so it seemed more natural to put them at the call site. That way they can also go into the corresponding L10n sub struct. Or are there any particular reasons to put them here?